### ISSUE TITLE
[Accessibility]: Profile/site-admin dropdown list items awkward to navigate to

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35313]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33515

### Problem description

From 2:45 in the recording.
Navigating to the button for the (red box) dropdown menu is fine. Once I click it, Im unable to navigate to the items in the dropdown with ctrl+option+right arrow, nor with tab. 
By typing ctrl+option+left arrow (which took me to the end of the top menu bar) and ctrl+option+right arrow, I was able to navigate through the dropdown items.

![image](https://user-images.githubusercontent.com/18282288/167920580-d160fb12-2bd0-4560-8189-fbba28da8f6f.png)

### Expected behavior

After clicking on the dropdown, ctrl+option+right arrow brings me to the first item in the dropdown list.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by umpox** [comment]
Partially intended behavior, this follows the [application menu](https://www.w3.org/WAI/tutorials/menus/application-menus/) pattern, which is preferred to allow quickly moving out of menu even if the user is half way down.

The main issue here is that the focus isn't correctly captured when the menu is opened. We have this issue for that: https://github.com/sourcegraph/sourcegraph/issues/36350, so will close this one.
